components/p3a/p3a_config.cc

```diff
@@ -75,21 +91,34 @@ inline void CheckURL(const GURL& url) {
 #endif  // !OFFICIAL_BUILD
 }
 
+std::string GetDefaultHost(const char* host_prefix) {
```

Comment:
why do all these methods take `const char*`?

Reply:
only constants are passed to this function